Index: serverside/cascadenik/openstreetmap/style.mml
===================================================================
--- serverside/cascadenik/openstreetmap/style.mml	(revision 448)
+++ serverside/cascadenik/openstreetmap/style.mml	(working copy)
@@ -4,14 +4,14 @@
 
 <!DOCTYPE Map[
     <!ENTITY map_srs "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +no_defs +over">
-    <!ENTITY coastlines "../../coastlines/processed_p">
+    <!ENTITY coastlines "coastlines/processed_p">
     <!ENTITY coastlines_srs "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +no_defs +over">
-    <!ENTITY osm_srs "+proj=latlong +ellps=WGS84 +datum=WGS84 +no_defs">
-    <!ENTITY host "127.0.0.1">
+    <!ENTITY osm_srs "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +no_defs +over">
+    <!ENTITY host "localhost">
     <!ENTITY port "5432">
-    <!ENTITY user "osm">
-    <!ENTITY password "0sm">
-    <!ENTITY dbname "osm_bayarea">
+    <!ENTITY user "postgres">
+    <!ENTITY password "">
+    <!ENTITY dbname "osm">
     <!ENTITY estimate_extent "false">
 ]>
 <Map srs="&map_srs;">
@@ -57,7 +57,7 @@
 
             <Parameter name="table"><![CDATA[
                 (SELECT *
-                 FROM osm_bayarea_polygon
+                 FROM planet_osm_polygon
                  WHERE building IS NULL
                    AND (amenity IN ('school', 'college', 'university', 'library', 'hospital', 'police', 'fire_station')
                         OR landuse IN ('industrial', 'commercial')
@@ -77,7 +77,7 @@
             <Parameter name="table"><![CDATA[
                 (SELECT *,
                         (CASE WHEN round(way_area * 1000000) >= 20 THEN 'large' WHEN round(way_area * 1000000) >= 1 THEN 'medium' ELSE 'small' END) AS size
-                 FROM osm_bayarea_polygon
+                 FROM planet_osm_polygon
                  WHERE building IS NULL
                    AND (amenity IN ('park')
                      OR landuse IN ('recreation_ground', 'greenfield', 'cemetery')
@@ -96,7 +96,7 @@
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
             <Parameter name="table"><![CDATA[
                 (SELECT *
-                 FROM osm_bayarea_polygon
+                 FROM planet_osm_polygon
                  WHERE building IS NULL
                    AND amenity = 'parking'
                  ORDER BY z_order ASC, way_area DESC) AS citylike]]></Parameter>
@@ -114,7 +114,7 @@
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
             <Parameter name="table"><![CDATA[
                 (SELECT *
-                 FROM osm_bayarea_polygon
+                 FROM planet_osm_polygon
                  WHERE landuse IN ('reservoir', 'water')
                     OR "natural" IN ('lake', 'water', 'land')
                     OR waterway IN ('canal', 'riverbank', 'river')
@@ -130,7 +130,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT * FROM osm_bayarea_line WHERE waterway IN ('river', 'stream', 'canal') ORDER BY z_order ASC) AS rivers</Parameter>
+            <Parameter name="table">(SELECT * FROM planet_osm_line WHERE waterway IN ('river', 'stream', 'canal') ORDER BY z_order ASC) AS rivers</Parameter>
         </Datasource>
     </Layer>
 
@@ -153,7 +153,7 @@
                               ELSE 4 END) AS priority,
                         (CASE WHEN highway IN ('secondary', 'trunk', 'primary', 'motorway', 'motorway_link') THEN 'major'
                               ELSE 'minor' END) AS prominence
-                 FROM osm_bayarea_line
+                 FROM planet_osm_line
                  WHERE highway IN ('residential', 'unclassified', 'service', 'minor', 'road', 'tertiary', 'secondary', 'trunk', 'primary', 'motorway', 'motorway_link')
                  ORDER BY z_order ASC, priority DESC)
                  
@@ -170,7 +170,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT * FROM osm_bayarea_line WHERE highway IN ('footpath', 'footway', 'steps', 'pedestrian', 'path', 'cycleway') ORDER BY z_order ASC) AS minor_roads</Parameter>
+            <Parameter name="table">(SELECT * FROM planet_osm_line WHERE highway IN ('footpath', 'footway', 'steps', 'pedestrian', 'path', 'cycleway') ORDER BY z_order ASC) AS minor_roads</Parameter>
         </Datasource>
     </Layer>
     <Layer class="road outline" srs="&osm_srs;">
@@ -191,7 +191,7 @@
                               ELSE 4 END) AS priority,
                         (CASE WHEN highway IN ('secondary', 'trunk', 'primary', 'motorway', 'motorway_link') THEN 'major'
                               ELSE 'minor' END) AS prominence
-                 FROM osm_bayarea_line
+                 FROM planet_osm_line
                  WHERE highway IN ('residential', 'unclassified', 'service', 'minor', 'road', 'tertiary', 'secondary', 'trunk', 'primary', 'motorway', 'motorway_link')
                  ORDER BY z_order ASC, priority DESC)
                  
@@ -208,7 +208,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT * FROM osm_bayarea_line WHERE highway IN ('footpath', 'footway', 'steps', 'pedestrian', 'path', 'cycleway') ORDER BY z_order ASC) AS minor_roads</Parameter>
+            <Parameter name="table">(SELECT * FROM planet_osm_line WHERE highway IN ('footpath', 'footway', 'steps', 'pedestrian', 'path', 'cycleway') ORDER BY z_order ASC) AS minor_roads</Parameter>
         </Datasource>
     </Layer>
     <Layer class="road inline" srs="&osm_srs;">
@@ -229,7 +229,7 @@
                               ELSE 4 END) AS priority,
                         (CASE WHEN highway IN ('secondary', 'trunk', 'primary', 'motorway', 'motorway_link') THEN 'major'
                               ELSE 'minor' END) AS prominence
-                 FROM osm_bayarea_line
+                 FROM planet_osm_line
                  WHERE highway IN ('residential', 'unclassified', 'service', 'minor', 'road', 'tertiary', 'secondary', 'trunk', 'primary', 'motorway', 'motorway_link')
                  ORDER BY z_order ASC, priority DESC)
                  
@@ -247,7 +247,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT * FROM osm_bayarea_line WHERE highway = 'motorway') AS motorways</Parameter>
+            <Parameter name="table">(SELECT * FROM planet_osm_line WHERE highway = 'motorway') AS motorways</Parameter>
         </Datasource>
     </Layer>
     <Layer class="motorway inline" srs="&osm_srs;">
@@ -259,7 +259,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT * FROM osm_bayarea_line WHERE highway = 'motorway') AS motorways</Parameter>
+            <Parameter name="table">(SELECT * FROM planet_osm_line WHERE highway = 'motorway') AS motorways</Parameter>
         </Datasource>
     </Layer>
 
@@ -272,7 +272,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT * FROM osm_bayarea_line WHERE railway IN ('rail', 'spur', 'subway', 'light_rail', 'tram', 'abandoned', 'disused', 'monorail') ORDER BY z_order ASC) AS railways</Parameter>
+            <Parameter name="table">(SELECT * FROM planet_osm_line WHERE railway IN ('rail', 'spur', 'subway', 'light_rail', 'tram', 'abandoned', 'disused', 'monorail') ORDER BY z_order ASC) AS railways</Parameter>
         </Datasource>
     </Layer>
 
@@ -285,7 +285,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT * FROM osm_bayarea_polygon WHERE building IS NOT NULL ORDER BY z_order ASC, way_area DESC) AS citylike</Parameter>
+            <Parameter name="table">(SELECT * FROM planet_osm_polygon WHERE building IS NOT NULL ORDER BY z_order ASC, way_area DESC) AS citylike</Parameter>
         </Datasource>
     </Layer>
 
@@ -299,7 +299,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT *, COALESCE(z_order, 0) AS correct_z_order, (CASE WHEN place = 'city' THEN 0 WHEN place = 'town' THEN 1 WHEN place = 'village' THEN 2 ELSE 3 END) AS priority FROM osm_bayarea_point WHERE place IN ('city', 'town', 'hamlet', 'village') ORDER BY priority ASC, correct_z_order DESC) AS places</Parameter>
+            <Parameter name="table">(SELECT *, COALESCE(z_order, 0) AS correct_z_order, (CASE WHEN place = 'city' THEN 0 WHEN place = 'town' THEN 1 WHEN place = 'village' THEN 2 ELSE 3 END) AS priority FROM planet_osm_point WHERE place IN ('city', 'town', 'hamlet', 'village') ORDER BY priority ASC, correct_z_order DESC) AS places</Parameter>
         </Datasource>
     </Layer>
     <Layer class="rail point" srs="&osm_srs;">
@@ -311,7 +311,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT *, y(astext(way)) AS latitude FROM osm_bayarea_point WHERE railway IN ('station', 'subway_entrance') AND name IS NOT NULL ORDER BY z_order ASC, latitude DESC) AS rail_points</Parameter>
+            <Parameter name="table">(SELECT *, y(astext(way)) AS latitude FROM planet_osm_point WHERE railway IN ('station', 'subway_entrance') AND name IS NOT NULL ORDER BY z_order ASC, latitude DESC) AS rail_points</Parameter>
         </Datasource>
     </Layer>
     <Layer class="road label major" srs="&osm_srs;">
@@ -327,7 +327,7 @@
                 (SELECT *, length(way) AS length, CHAR_LENGTH(ref) AS ref_length,
                         (CASE WHEN CHAR_LENGTH(ref) BETWEEN 2 AND 9 THEN ref ELSE NULL END) AS ref_content,
                         (CASE WHEN highway IN ('motorway', 'motorway_link') THEN 0 WHEN highway IN ('trunk', 'primary') THEN 1 WHEN highway = 'secondary' THEN 2 WHEN highway = 'tertiary' THEN 3 ELSE 4 END) AS priority
-                 FROM osm_bayarea_line
+                 FROM planet_osm_line
                  WHERE highway IN ('secondary', 'trunk', 'primary', 'motorway', 'motorway_link')
                  ORDER BY priority ASC, z_order ASC, length DESC) AS major_roads]]></Parameter>
         </Datasource>
@@ -347,7 +347,7 @@
                               WHEN round(way_area * 1000000) >= 1 THEN 'medium'
                               ELSE 'small' END) AS size,
                         REGEXP_REPLACE(UPPER(name), '(.)', E'\\1\xC2\xA0\xC2\xA0', 'g') AS name_stretched
-                 FROM osm_bayarea_polygon
+                 FROM planet_osm_polygon
                  WHERE building IS NULL
                    AND (amenity IN ('park')
                      OR landuse IN ('recreation_ground', 'greenfield', 'cemetery')
@@ -370,7 +370,7 @@
                               WHEN round(way_area * 1000000) >= 1 THEN 'medium'
                               ELSE 'small' END) AS size,
                         REGEXP_REPLACE(UPPER(name), '(.)', E'\\1\xC2\xA0\xC2\xA0', 'g') AS name_stretched
-                 FROM osm_bayarea_polygon
+                 FROM planet_osm_polygon
                  WHERE building IS NULL
                    AND (amenity IN ('parking', 'school', 'college', 'university', 'library', 'hospital', 'police', 'fire_station')
                      OR landuse IN ('industrial', 'commercial')
@@ -387,7 +387,7 @@
             <Parameter name="password">&password;</Parameter>
             <Parameter name="dbname">&dbname;</Parameter>
             <Parameter name="estimate_extent">&estimate_extent;</Parameter>
-            <Parameter name="table">(SELECT *, length(way) AS length, (CASE WHEN highway IN ('motorway', 'motorway_link') THEN 0 WHEN highway IN ('trunk', 'primary') THEN 1 WHEN highway = 'secondary' THEN 2 WHEN highway = 'tertiary' THEN 3 ELSE 4 END) AS priority FROM osm_bayarea_line WHERE highway IN ('residential', 'unclassified', 'service', 'minor', 'road', 'tertiary') ORDER BY priority ASC, z_order ASC, length DESC) AS minor_roads</Parameter>
+            <Parameter name="table">(SELECT *, length(way) AS length, (CASE WHEN highway IN ('motorway', 'motorway_link') THEN 0 WHEN highway IN ('trunk', 'primary') THEN 1 WHEN highway = 'secondary' THEN 2 WHEN highway = 'tertiary' THEN 3 ELSE 4 END) AS priority FROM planet_osm_line WHERE highway IN ('residential', 'unclassified', 'service', 'minor', 'road', 'tertiary') ORDER BY priority ASC, z_order ASC, length DESC) AS minor_roads</Parameter>
         </Datasource>
     </Layer>
     <Layer class="water label" srs="&osm_srs;">
@@ -405,7 +405,7 @@
                               WHEN round(way_area * 1000000) >= 1 THEN 'medium'
                               ELSE 'small' END) AS size,
                         REGEXP_REPLACE(UPPER(name), '(.)', E'\\1\xC2\xA0\xC2\xA0', 'g') AS name_stretched
-                 FROM osm_bayarea_polygon
+                 FROM planet_osm_polygon
                  WHERE landuse IN ('reservoir', 'water')
                     OR "natural" IN ('lake', 'water', 'land')
                  ORDER BY z_order ASC) AS water]]></Parameter>
@@ -426,7 +426,7 @@
                               WHEN round(way_area * 1000000) >= 1 THEN 'medium'
                               ELSE 'small' END) AS size,
                         REGEXP_REPLACE(UPPER(name), '(.)', E'\\1\xC2\xA0\xC2\xA0', 'g') AS name_stretched
-                 FROM osm_bayarea_polygon
+                 FROM planet_osm_polygon
                  WHERE building IS NOT NULL
                  ORDER BY z_order ASC, way_area DESC) AS citylike]]></Parameter>
         </Datasource>
